### PR TITLE
Make the archived artifacts summary on build pages theme-aware

### PR DESCRIPTION
**Pull request by VirtualTim**
Before:
image

After:
image

Light theme is unchanged.

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
AFAIK it will require a fallback property for browsers which do not support CSS variables. e.g. we still support IE 11 until we modify the browser support policy. Please correct me if I am wrong @timja @fqueiruga 

Also, I wonder whether we want to backport such fixes into 2.249.x LTS in order to facilitate adoption of the new theming engine

**Review by fqueiruga — CHANGES_REQUESTED** [review]
I suggest not changing the YUI files directly. We have one css for YUI CSS overrides: https://github.com/jenkinsci/jenkins/blob/master/war/src/main/less/base/yui-compatibility.less

Can you please add entries with the fixes for these selectors to that file and leave treeview-skin.css unchanged?

**Comment by fqueiruga** [comment]
It would require a fallback indeed.
Also, I'd vote against backporting this. It is not a regression in Jenkins behaviour and it's just to support experimental changes.

**Comment by VirtualTim** [comment]
Unless I'm missing something, yui-compatibility.less already uses css variables. Does that get transpiled into something that works with IE11?

**Comment by timja** [comment]
If you put your code into less files yes it gets transpiled and will work for IE11. The css files aren’t run through webpack / postcss

**Comment by daniel-beck** [comment]
There's an actual tree view (nested views) in Jenkins, experimental and hidden behind a flag, so changed the title to disambiguate.

**Review by timja — APPROVED** [review]

File: war/src/main/less/base/yui-compatibility.less
Comment (fqueiruga) [inline war/src/main/less/base/yui-compatibility.less:98]: I'd put a comment in this section explaining what it's for

File: war/src/main/less/base/yui-compatibility.less
Comment (fqueiruga) [inline war/src/main/less/base/yui-compatibility.less:99]: Not a blocker, but could you try using on the greys used on the palette? You can look at that on the theme.less file.

**Review by fqueiruga — APPROVED** [review]
Approved with minor comments

File: war/src/main/less/base/yui-compatibility.less
Comment (VirtualTim) [reply war/src/main/less/base/yui-compatibility.less:99]: I picked `--panel-header-bg-color` because I think that every theme will set that to something that contrasts with the background colour.
Did you have a specific colour in mind?

File: war/src/main/less/base/yui-compatibility.less
Comment (VirtualTim) [reply war/src/main/less/base/yui-compatibility.less:100]: Done. Is this the kind of thing you meant?

File: war/src/main/less/base/yui-compatibility.less
Comment (fqueiruga) [reply war/src/main/less/base/yui-compatibility.less:101]: I meant for the fallback but now that I think of it it's not needed

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

File: war/src/main/less/base/yui-compatibility.less
Comment (VirtualTim) [reply war/src/main/less/base/yui-compatibility.less:101]: Yeah I just kept the fallback the old value.

**Pull request merged**
